aixcalibuha/calibration/calibrator.py

```diff
@@ -397,6 +398,15 @@ def _kpi_and_logging_calculation(self, *, xk_descaled, counter, results):
                 # Changed to false in this script after calling function "save_calibration_results"
                 "Penaltyfactor": penalty
             }
+            if self.save_current_best_iterate:
+                parameter_values = self._get_parameter_dict_from_current_best_iterate()
+                
+                temp_save = {
+                    "parameters": parameter_values,
+                    "objective": total_res
+                }
+                with open(self.working_directory / 'best_iterate.json', 'w') as json_file:
```

Comment:
Use `self.working_directory.joinpath("best_iterate.json")`?

Reply:
/ is equivalent to joinpath and in my opinion better readable for chained operations. But changed it anyway to joinpath, because this is mainly used in the rest of the code.